USAGE_android.md

```diff
@@ -102,8 +213,135 @@ It may also be set as a post attach command in the project configuration:
 * Enter the `process handle SIGSEGV -n true -p true -s false` command
 * Click "Apply"
 
+
+### To Root Or Not To Root
+
+Rooting the Android device for capture is not required if you are attempting
+to capture an application you can build in Debug mode and associate it with the
+GfxReconstruct layer during the build.
+However, if you can not do that, your only option is to attempt to capture
+the image on a rooted device.
+Instructions for rooting Android devices are available on the internet, but we
```

Comment:
I think you can omit "but we won't reference any instructions here".  Maybe "Instructions for rooting Android devices vary and are available through an internet search."